src/appkit.cs

```diff
@@ -25201,4 +25769,278 @@ interface NSSliderAccessoryBehavior : NSCoding, NSCopying
 		[Export ("handleAction:")]
 		void HandleAction (NSSliderAccessory sender);
 	}
+
+	[Mac (10,13)]
+	[BaseType (typeof(NSObject))]
+	interface NSAccessibilityCustomAction
+	{
+		[Export ("initWithName:handler:")]
+		IntPtr Constructor (string name, [NullAllowed] Func<bool> handler);
+
+		[Export ("initWithName:target:selector:")]
+		IntPtr Constructor (string name, NSObject target, Selector selector);
+
+		[Export ("name")]
+		string Name { get; set; }
+
+		[NullAllowed, Export ("handler", ArgumentSemantic.Copy)]
+		Func<bool> Handler { get; set; }
+
+		[NullAllowed, Export ("target", ArgumentSemantic.Weak)]
+		NSObject Target { get; set; }
+
+		[Advice (@"It must conform to one of the following signatures: 
+		- (BOOL)myPerformActionMethod;
+		- (BOOL)myPerformActionMethod:(NSAccessibilityCustomAction *)action;")]
```

Comment:
Maybe we could use C# signature notation instead like

`bool MyPerformAction ()` or `bool MyPerformAction (NSAccessibilityCustomAction)`

They do need to know that they need to decorate any of those signatures with the corresponding `ExportAttribute`